src/core/expr/fexpr_fillna.cc

```diff
@@ -55,9 +55,9 @@ class FExpr_FillNA : public FExpr_Func {
         out +=", value=";
         out += value_->repr();
       } else {
-        out += ", reverse=";
-        out += reverse_? "True" : "False";
-      }
+          out += ", reverse=";
+          out += reverse_? "True" : "False";
+        }
```

Comment:
Why do we need this indent?

Reply:
this section is under the `else` section, hence the indent so the `}` aligns under `else`. I have restored it to the previous form, however, i thought that was the idea behind indent, so any reader (me inclusive) can better understand the code (which section refers to if, else, etc, readability)